app/serializers/api/admin/for_order_cycle/enterprise_serializer.rb

```diff
@@ -6,7 +6,11 @@ class Api::Admin::ForOrderCycle::EnterpriseSerializer < ActiveModel::Serializer
              :is_primary_producer, :is_distributor, :sells
 
   def issues_summary_supplier
-    issues = OpenFoodNetwork::EnterpriseIssueValidator.new(object).issues_summary confirmation_only: true
+    issues =
+      OpenFoodNetwork::EnterpriseIssueValidator.
+        new(object).
+        issues_summary(confirmation_only: true)
```

Comment:
I experience this sometimes and it feels like making the code less readable just to pass the rules we use for Rubocop. But I have no suggestion that would make styling existing code easier. :+1: